src/server.c

```diff
@@ -574,6 +574,54 @@ null_log_writer (GLogLevelFlags   log_level,
   return G_LOG_WRITER_HANDLED;
 }
 
+/* Bind server to available IPv4 and IPv6 local addresses
+ *
+ * The server must be NOT listening on any interface.
+ *
+ * If port is 0, the server will find an unsued port to listen on. The
```

Comment:
nit:  s/unsued/unused/

Reply:
Indeed. No ports were sued in this PR...